parsing/lexer.mll

```diff
@@ -344,10 +344,12 @@ let symbolchar =
   ['!' '$' '%' '&' '*' '+' '-' '.' '/' ':' '<' '=' '>' '?' '@' '^' '|' '~']
 let dotsymbolchar =
   ['!' '$' '%' '&' '*' '+' '-' '/' ':' '=' '>' '?' '@' '^' '|']
+let letopsymbolchar = dotsymbolchar | ['.' '\\' 'a'-'z' 'A'-'Z' '_' '0'-'9']
```

Comment:
The character `\` is only used for escape sequence, it seems more coherent to not include it here.